docs/hugo/content/reference/_index.md

```diff
@@ -16,268 +16,268 @@ grouped by the originating ARM service.
 
 ## Authorization
 
-| Resource       | ARM Version        | CRD Version                | Supported From | Sample                                                                                                                                           |
-|----------------|--------------------|----------------------------|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------|
-| RoleAssignment | 2020-08-01-preview | v1beta20200801preview      | v2.0.0-beta.0  | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples/authorization/v1beta20200801preview_roleassignment.yaml)      |
-| RoleAssignment | 2020-08-01-preview | v1alpha1api20200801preview | v2.0.0-alpha.2 | [View](https://github.com/Azure/azure-service-operator/tree/main/v2/config/samples/authorization/v1alpha1api20200801preview_roleassignment.yaml) |
+| Resource       | ARM Version        | CRD Version                | Supported From | Sample                                                                                                                                                      |
+|----------------|--------------------|----------------------------|----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| RoleAssignment | 2020-08-01-preview | v1beta20200801preview      | v2.0.0-beta.0  | [View](https:/github.com/Azure/azure-service-operator/tree/main/v2/config/samples/authorization/v1beta/v1beta20200801preview_roleassignment.yaml)           |
```

Comment:
These URLs are malformed - need `http://` not just `http:/`.

Reply:
Good catch! Thanks